charts/kube-prometheus-stack/README.md

```diff
@@ -163,26 +161,6 @@ You can read more information on how to add firewall rules for the GKE control p
 
 Alternatively, you can disable the hooks by setting `prometheusOperator.admissionWebhooks.enabled=false`.
 
-### Helm fails to create CRDs
```

Comment:
[Helm 3 will only install CRDs and not perform any updates on them afterwards, any changes to CRDs still require someone to maintain them outside of helm](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you). Do you think we should move this section into the upgrade instructions?

Reply:
Yes we should, if there were past versions of the CRDs, in the upgrading section for the chart version where the new CRD versions were introduced, we should note that they need to be manually upgraded to that newer version 👍

Reply:
@mrueg can you accept the requested changes so we can proceed with merging the Pull Request?

Reply:
@r3kzi there's nothing I can accept from a Github UI, I added a paragraph to the update piece linking the doc that @vsliouniaev  mentioned.